ReactiveFeedback/FeedbackLoop.swift

```diff
@@ -0,0 +1,182 @@
+import ReactiveSwift
+
+public final class FeedbackLoop<State, Event> {
```

Comment:
@andersio I've removed conformance to `PropertyProtocol` not sure why we need it?
Because it also gonna crash if you try to access the value before starting it.

Reply:
oops